modules/ROOT/pages/connections.adoc

```diff
@@ -4,33 +4,32 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, sdk, connection, connectivity, management, pooling, cached, provider, connection-provider
 
-Connected modules (connectors) are probably the most common type of module.
-
-The connectivity framework starts with the concept of a `ConnectionProvider`.
-This is an interface responsible for creating, disconnecting, and describing the connection so that the runtime can handle it properly, while remaining abstracted from the concerns of managing those connections.
+Connected modules (connectors) are the most common type of module. They provide a connection between a Mule flow and an external resource.
+Configure these connections with the `ConnectionProvider` in a connectivity framework to create, disconnect, and describe the connection for Mule runtime engine to handle.
+For more information on how to use connections, refer to <<use-connections, this section>>.
 
 == Declaring a Connection Provider
 
 To declare a new `ConnectionProvider` you should create a new class that implements the `ConnectionProvider` interface. The `ConnectionProvider` interface is a typed interface that receives one type argument that indicates the class of the instances that the `ConnectionProvider` handles.
 
-This example declares a `FooConnectionProvider` class that implements the `ConnectionProvider` interface:
+This example declares a `myConnectionProvider` class that implements the `ConnectionProvider` interface:
```

Comment:
by naming convention, class name should start with capital letter, so I'd replace _myConnectionProvider_ with _MyConnectionProvider_